packages/extension/src/components/pages/AddNewTrustline/AddNewTrustline.tsx

```diff
@@ -259,14 +325,22 @@ export const AddNewTrustline: FC = () => {
           </>
         }
         transaction={TransactionStatus.Rejected}
+        {...(params.inAppCall ? { onClick: () => navigate(HOME_PATH) } : {})}
```

Comment:
Is there a use case when you would be inApp and not have some parameters? I would remove this line.

Reply:
During the first microseconds of rendering, as far as I recall

Reply:
Done